guppylang/decorator.py

```diff
@@ -492,11 +492,12 @@ def _parse_expr_string(ty_str: str, parse_err: str, sources: SourceMap) -> ast.e
             sources.add_file(info.filename)
             source_lines, _ = inspect.getsourcelines(caller_module)
             source = "".join(source_lines)
-            annotate_location(expr_ast, source, info.filename, 0)
+            annotate_location(expr_ast, source, info.filename, 1)
```

Comment:
Drive-by: Our new spans start counting lines at 1

Reply:
Columns also start at 1, but that's the same in Python AST afaik, so it works out without having to modify it